app/queries/consumer_apps/all_query.rb

```diff
@@ -0,0 +1,14 @@
+module ConsumerApps
+  class AllQuery
+    def self.call
+      forem_bundle = ConsumerApp::FOREM_BUNDLE
+      existing_platforms = ConsumerApp.where(app_bundle: forem_bundle).pluck(:platform)
+      (ConsumerApp::FOREM_APP_PLATFORMS - existing_platforms).each do |platform|
+        # Re-create the supported Forem apps if they're missing
+        ConsumerApp.create(app_bundle: forem_bundle, platform: platform, active: true)
+      end
+
+      ConsumerApp.all.limit(50)
```

Comment:
```suggestion
      ConsumerApp.limit(50)
```

should they be ordered somehow or just any 50 apps?